### DISCUSSION TITLE
Task list as a seperate section so no need to scroll down long descriptions to edit

### DISCUSSION THREAD

**Discussion by BohoCode** [community/community#5375] [Projects and Issues]
In an issue with a long description with a task list at the bottom it quickly becomes tedious to edit the description and then go and find the use created **tasks** section. It feels like should be a citizen in it's own right with a quick link to edit an issues associated tasks

**Comment by deeno35** [comment]
It's funny, I came here to submit feedback because I was thinking that the task list should show up above the commit message when creating PR's instead of the bottom.  It would also be nice if I didn't have to put a separator into the template myself.  Overall this idea of a separate section is way better.  It seems like it really belongs in the checks tab or something similar
